setup.cfg

```diff
@@ -0,0 +1,7 @@
+[flake8]
```

Comment:
Not sure this is an appropriate addition at this time.

Reply:
Turns out this is actually required. `W503` and `W504` are the opposites of each other and at least one must be selected to satisfy flake8 when dealing with long lines with binary operators.